sqlmesh/core/model/kind.py

```diff
@@ -277,6 +286,26 @@ class FullKind(_ModelKind):
     name: Literal[ModelKindName.FULL] = ModelKindName.FULL
 
 
+class SCDType2Kind(_ModelKind):
+    name: Literal[ModelKindName.SCD_TYPE_2] = ModelKindName.SCD_TYPE_2
+    unique_key: t.List[str]
+    valid_from_name: str = "valid_from"
```

Comment:
maybe valid_from_column etc

Reply:
Actually I'm not sure since one distinct aspect of these parameters is that these are names that we will use when creating the table and not columns that are represented in the SQL itself. So for example `time_column` maps to a column in your SQL so calling this `valid_from_column` matches that pattern and could leave an impression it should be defined in the SQL. Breaking that pattern with `name` can help prevent a user from associating the two (since they are fundamentally different).

Reply:
ok